script.js

```diff
@@ -0,0 +1,62 @@
+let bookList = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author) {
+  bookList.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(bookList));
+}
+
+function removeBook(element, title, author) {
```

Comment:
- [ ] The **_removeBook()_** method is not working as intended. When the same book title and book author are entered twice or more and subsequently one of them is deleted, both all entries add with the same title and the author gets removed as well. We actually want each entry to have a **unique id**, or perhaps you can improve upon this aspect by **preventing duplicates**